aspnet/fundamentals/error-handling.rst

```diff
@@ -1,8 +1,181 @@
-.. include:: /../common/stub-topic.txt
+Error Handling
+==============
+By `Steve Smith`_
 
-|stub-icon| Error Handling
-==========================
+When errors occur in your ASP.NET app, you can handle them in a variety of ways, as described in this article.
 
-.. include:: /../common/stub-notice.txt
+.. contents:: Sections
+	:local:
+	:depth: 1
+	
+`View sample files <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/error-handling/sample>`_
```

Comment:
This text for linkign to the sample needs to be consistent across all of our documents:

```
`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/error-handling/sample>`__
```